run/o2simdefaultdetectorlist.json

```diff
@@ -11,6 +11,7 @@
     "MCH",
     "MID",
     "ZDC",
+    "FOC",
```

Comment:
FOCal is not part of ALICE2, no?

Reply:
I agree. FOCAL should probably be in the ALICE2.1 detector list.

Reply:
Already is :)

Reply:
Sorry I didn't see it in ALICE2.1.

Reply:
np